.pre-commit-config.yaml

```diff
@@ -66,7 +67,7 @@ repos:
       rev: v0.9.0.6
       hooks:
           - id: shellcheck
-            args: [--external-sources]
+            args: [--exclude=SC2155, --external-sources]
```

Comment:
back at it again

![image](https://github.com/user-attachments/assets/da1fdc07-69fb-4ded-a720-35108d4a6374)